my_list.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'my_enumerable'
+class MyList
+  include MyEnumerable
+
+  def initialize(*items)
+    @list = items
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+# list.my_filter {|e| e.even?}
+
+# list.my_any? {|e| e == 2}
+
+# list.my_any? {|e| e == 5}
```

Comment:
- [ ] It's against the best practices to leave un-used code as a comment on your code base, I'll suggest you remove this. :smile: